app/components/error_summary_component/view.rb

```diff
@@ -0,0 +1,8 @@
+module ErrorSummaryComponent
+  class View < ViewComponent::Base
```

Comment:
```suggestion
  class View < ViewComponent::Base
  
  attr_accessor :errors
```

then in the view we can use `errors` rather than `@errors` which a dev may assume has come from a controller action etc.